### PR TITLE
Add signal service

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

**Comment by kasperl** [comment]
The intent is that you can import the `cellular` package and use it like this:

```
import cellular.signal

main:
  quality := signal.quality
  print "power = $quality.power"
```

It feels like I'll need a different name for `SignalQuality` seeing how the quality is only one part of it.

File: src/signal.toit [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2023 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import .api.signal
+
+service_/SignalServiceClient? ::= (SignalServiceClient).open
+    --if_absent=: null
+
+class SignalQuality:
```
Comment (floitsch) [inline src/signal.toit:10]: `SignalMetrics` or `SignalInfo` ?

File: src/signal.toit [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  power/float?
```
Comment (floitsch) [inline src/signal.toit:11]: It seems like this is typically called "strength".

**Review by floitsch — APPROVED** [review]
LGTM.

**Pull request merged**